microprofile.ls/org.eclipse.lsp4mp.ls/src/test/resources/org/eclipse/lsp4mp/extensions/reactivemessaging/mp-reactive-messaging-properties.json

```diff
@@ -0,0 +1,80 @@
+{
+	"properties": [
+		{
+			"type": "org.eclipse.microprofile.reactive.messaging.spi.Connector",
+			"sourceMethod": "process(I)D",
+			"required": true,
+			"phase": 0,
+			"name": "mp.messaging.incoming.prices.connector",
+			"sourceType": "org.acme.kafka.PriceConverter",

```

Comment:
This also shouldn't be hardcoded

Reply:
What is the problem with that? It's just for test.

Reply:
Okay. Sorry, I couldn't tell that this was just for the tests.